mule-extensions-api/src/test/java/org/mule/runtime/extension/api/dsl/model/ComplexFieldsType.java

```diff
@@ -35,6 +36,10 @@
   @NullSafe
   private NotGlobalType notGlobalType;
 
+  @Parameter
+  @Content
+  private String fieldAsContent;
```

Comment:
what is going to happen with this? didn't you just made it so that a field cannot be `@Content`?